closed/autoconf/custom-spec.gmk.in

```diff
@@ -85,6 +85,10 @@ OPENJ9_PLATFORM_CODE    := @OPENJ9_PLATFORM_CODE@
 
 OPENJ9_LIBS_SUBDIR      := @OPENJ9_LIBS_SUBDIR@
 
+# Mixed References Mode
+OPENJ9_ENABLE_MIXED_REFERENCES := @OPENJ9_ENABLE_MIXED_REFERENCES@
+OMR_MIXED_REFERENCES_MODE   := @OMR_MIXED_REFERENCES_MODE@
```

Comment:
I don't think we need both of these, if `OMR_MIXED_REFERENCES_MODE` is one of `off` (or something similar), `static` or `dynamic`.